src/sweeps/config/schema.json

```diff
@@ -753,6 +753,10 @@
       "type": "string",
       "description": "Launch Job to run"
     },
+    "docker_image_uri": {
```

Comment:
might want to make this a subkey of a `launch_scheduler` section or something, and also include a key for `num_workers`